Source/Core/ReferencePointGeometryInstanceAttribute.js

```diff
@@ -0,0 +1,150 @@
+define([
+        './Cartesian3',
+        './Cartographic',
+        './Check',
+        './ComponentDatatype',
+        './defined',
+        './defineProperties',
+        './EncodedCartesian3',
+        './Matrix4',
+        './Transforms'
+    ], function(
+        Cartesian3,
+        Cartographic,
+        Check,
+        ComponentDatatype,
+        defined,
+        defineProperties,
+        EncodedCartesian3,
+        Matrix4,
+        Transforms) {
+    'use strict';
+
+    /**
+     * Batch table attribute representing the HIGH or LOW bits of an EncodedCartesian3.
+     *
+     * @param {Cartesian3} vec3 HIGH or LOW bits of an EncodedCartesian3
+     * @private
```

Comment:
Will users using the primitive API directly ever want to use this?

Reply:
The use case for this is pretty specific, I'm just going to build this functionality into `ClassificationPrimitive`.